lib/hawkular/alerts/alerts_api.rb

```diff
@@ -81,12 +81,100 @@ def create_trigger(trigger, conditions = [], dampenings = [], _actions = [])
       http_post 'triggers/trigger', full_trigger
     end
 
+    # Creates the group trigger definition.
+    # @param [Trigger] trigger The group trigger to be created
+    # @return [Trigger] The newly created group trigger
+    def create_group_trigger(trigger)
+      ret = http_post 'triggers/groups', trigger.to_h
+      Trigger.new(ret)
+    end
+
+    # Updates a given group trigger definition
+    # @param [String] trigger_id ID of the group trigger to update
+    # @param [Trigger] trigger the group trigger to be updated
+    # @return [Trigger] The updated group trigger
+    def update_group_trigger(trigger_id, trigger)
```

Comment:
It could be. This method is mapping with the REST one defined at
https://github.com/hawkular/hawkular-alerts/blob/master/hawkular-alerts-rest/src/main/java/org/hawkular/alerts/rest/TriggersHandler.java#L488
so, I find it more interesting to maintain same parameter.